lib/modules/manager/crossplane/extract.ts

```diff
@@ -0,0 +1,49 @@
+import { loadAll } from 'js-yaml';
+import { logger } from '../../../logger';
+import { getDep } from '../dockerfile/extract';
+import type {
+  ExtractConfig,
+  PackageDependency,
+  PackageFileContent,
+} from '../types';
+import { XPKGSchema } from './schema';
+
+export function extractPackageFile(
+  content: string,
+  packageFile: string,
+  extractConfig?: ExtractConfig,
+): PackageFileContent | null {
+  let list = [];
+  try {
+    list = loadAll(content);
```

Comment:
we should short circuit here and check if the file contains the crossplain resources. a simple `string.contains` should be good enough.

this way we can improve performance via skipping yaml parsing and zod validation.

Reply:
done 👍